src/visual.ts

```diff
@@ -1157,7 +1157,7 @@ module powerbi.extensibility.visual {
                         y = yScale(dataPoint.y);
                         dy = dataPoint.size.height;
 
-                        return y - dy;
+                        return y - dy < 0 ? y + dy / 2 : y - dy;
```

Comment:
Please add a variable that contains `y - dy`. Don't see any reason to compute it twice